### PR TITLE
W6-SurveyForm

### PR DESCRIPTION
**Pull request by Maria-Manuela**
## Netlify link
https://surveyformbyam.netlify.app/

## Collaborators
[github-vidalhuix]

### CODE REVIEW

**Review by FridaMari — COMMENTED** [review]
Nice app, especially the start, next and back buttons! :) When reading the code it was a bit confusing with the overall structure. Seems like there are some file duplicates.

File: src/Summary.jsx
Comment (AntonellaMorittu) [inline src/Summary.jsx:1]: pls format this file to remove empty lines on top of the file 🧹

File: src/Survey.jsx
Comment (AntonellaMorittu) [inline src/Survey.jsx:1]: please format all the files in the app and set the correct n of tabs, you can check your Toolbox on Notion how to set up your Visual Studio Code 👍

File: src/Footer.jsx
Comment (AntonellaMorittu) [inline src/Footer.jsx:1]: remove the double files, these components (Footer, Header, Summary, Survey) already exist in the components folder, so go ahead and remove them from here 👍

File: src/assets/Technigo-logo-1.png
Comment (AntonellaMorittu) [inline src/assets/Technigo-logo-1.png:1]: remove all unused assets🧹

File: src/components.survey/Survey.css
Comment (AntonellaMorittu) [inline src/components.survey/Survey.css:1]: format format format 🪄

**Review by AntonellaMorittu — CHANGES_REQUESTED** [review]
Hi Maria-Manuela, congratulations with your Survey app! ⭐️ You developed a great app with multi-step form and it features all the needed requirements. 
Here's what needs to be improved:
- good practice: the structure can be improved removing the double components inside the `src` folder. Make sure to name the `components` folder just like so
- formatting needs to be applied for both `.jsx`jsx and `.css` files, please check our guide on Notion [here](https://www.notion.so/technigo/How-to-use-VS-Code-5ed620b8e8f24f17bcab6e285958d9b6?pvs=4#a58443dde71c4918a030dc6de3cd62cd) so that from now on all projects will be formatted on save and have the right two-space tab indentation :) Thank you 💪

**Review by AntonellaMorittu — APPROVED** [review]

**Pull request closed without merging**